src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.cs

```diff
@@ -128,6 +128,13 @@ internal int EffectiveMaxConcurrencyLevel
     /// </remarks>
     public static partial class Parallel
     {
+#if TARGET_BROWSER
+        internal const bool IsSingleThreadedHost = true;
+#else
+// FIXME: The ifdef doesn't work
+        internal const bool IsSingleThreadedHost = true;
```

Comment:
This needs to be changed :-)

Reply:
yes, as mentioned in the description, something is wrong with the build system so i can't get the ifdef to work. if anyone has any ideas i'd love to try them, i probably spent an hour trying

Reply:
The project doesn't have a browser specific version in `TargetFrameworks`.

Reply:
Note that it would probably be just as effective to use `OperatingSystem.IsBrowser` and `SupportedOSPlatformGuard` attribute. The linker will eliminate it.